src/pages/Health.tsx

```diff
@@ -1,13 +1,141 @@
-import {Box, Heading} from '@chakra-ui/react';
+import {ExternalLinkIcon} from '@chakra-ui/icons';
+import {
+  Box,
+  Heading,
+  Text,
+  Link,
+  UnorderedList,
+  ListItem,
+  Image,
+  Flex,
+} from '@chakra-ui/react';
 import React from 'react';
 import {useTranslation} from 'react-i18next';
+import AqiTable from '../components/AqiTable';
 
 const Health: React.FC = () => {
   const {t} = useTranslation('health');
+
   return (
-    <Box>
-      <Heading>{t('heading')}</Heading>
-    </Box>
+    <Flex width="full" align="center" direction="column" padding={8}>
+      <Heading as="h1">{t('pageHeading')}</Heading>
+      <Flex
+        padding={8}
+        margin={8}
+        width="full"
+        borderWidth={1}
+        borderRadius={8}
+        boxShadow="lg"
+        direction="column"
+      >
+        <Heading>{t('pollution.heading')}</Heading>
+        <Text>{t('pollution.paragraph1')}</Text>
+        <Flex
+          padding={4}
+          margin={['2', 'auto', '2', 'auto']}
+          borderWidth={1}
+          borderRadius={8}
+          boxShadow="lg"
+          width={['100%', null, '525px', null]}
+        >
+          <Flex direction="column" align="center">
+            <Image
+              src={t('pollution.image.fileName')}
+              alt={t('pollution.image.caption')}
+              maxWidth={['95%', null, '500px', null]}
```

Comment:
Super nit: I think that 500px is just enough that the `link to external webpage` icon goes onto the next line, consider increasing to 520 or something so that it's all on one line instead of going to a second line

Reply:
I'm a bit confused by what is going on here. I think this might be something that is easier to talk through on Zoom tomorrow because I'm not understanding this comment.

Reply:
Ignoring due to development time vs. effect per Zoom discussion